scripts/azureml-assets/azureml/assets/validate_assets.py

```diff
@@ -749,14 +750,34 @@ def validate_model_spec(asset_config: assets.AssetConfig) -> int:
         _log_error(asset_config.file_name_with_path, f"{MLFlowModelTags.LICENSE} missing")
         error_count += 1
 
+    # shared compute check
     if MLFlowModelTags.SHARED_COMPUTE_CAPACITY not in model.tags:
-        _log_error(asset_config.file_name_with_path, f"{MLFlowModelTags.SHARED_COMPUTE_CAPACITY} missing")
+        _log_error(asset_config.file_name_with_path, f"Tags {MLFlowModelTags.SHARED_COMPUTE_CAPACITY} missing")
+        error_count += 1
+
+    if not model.properties.get(MLFlowModelProperties.SHARED_COMPUTE_CAPACITY, False):

```

Comment:
You want MLFlowModelProperties.SHARED_COMPUTE_CAPACITY to be in model.properties and its value to be True (or any truthy value, like 1)?

Reply:
this is an example in yaml:

```
properties:
  SharedComputeCapacityEnabled: true
  SHA: ef35fe7767c1dad71f5c853838cdd80d0b3441ed

```